config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+ 
+
+  root 'static#index' # assuming 'static#index' is your static view
+  # get '/api/random_greeting', to: 'greetings#random', as: 'random_greeting'
+  # get 'greetings/random', to: 'greetings#random'
+  # Defines the root path route ("/")
+  # root "posts#index"
```

Comment:
- [ ] @chudisoft Kindly remove the commented code, its not good practise having commented code in the codebase.